### PR TITLE
Support Organizations that do not use AWS Control Tower

### PR DESCRIPTION
**Pull request by kkvinjam**

### CODE REVIEW

**Comment by kkvinjam** [comment]
> E2E test has completed with errors. If you are an external contributor, please contact the project maintainers for more information.

The failure was due to a test environment issue.

File: .project_automation/functional_tests/entrypoint.sh [hunk 1]
```diff
@@ -5,6 +5,8 @@
 # managed and local tasks always use these variables for the project and project type path
 PROJECT_PATH=${BASE_PATH}/project
 PROJECT_TYPE_PATH=${BASE_PATH}/projecttype
+NON_CT_ENV="039084729647"
```
[the remark this answers is not in the document]
Comment (kkvinjam) [reply .project_automation/functional_tests/entrypoint.sh:8]: Yes, this is an interim solution until a test solution is implemented to test in both CT and non-CT environments.

File: .taskcat.yml [hunk 2]
```diff
@@ -28,4 +28,25 @@ tests:
...
+      pSRAS3BucketRegion: us-east-1
+      pAutoEnableS3Logs: 'true'
+      pAutoEnableKubernetesAuditLogs: 'false'
+      pAutoEnableMalwareProtection: 'false'
+      pDeepwatchRoleName: 'deepwatch-mdr-role'
+      pEnableLambdaDataEvents: 'false'
+      pEnableS3DataEvents: 'true'
+      pDisableGuardDuty: 'Yes'
+      pControlTower: 'false'
+      pLogArchiveAccountId: '866786363886'
```
[the remark this answers is not in the document]
Comment (kkvinjam) [reply .taskcat.yml]: Fixed. Replaced with SSM parameters.

**Review by gcasilva — APPROVED** [review]

**Pull request merged**